pkgs/applications/audio/osmid/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, alsaLib
+, libX11
+}:
+
+stdenv.mkDerivation rec {
+  name = "osmid";
+  version = "0.6.8";
+
+  src = fetchFromGitHub {
+    owner = "llloret";
+    repo = "osmid";
+    rev = "v${version}";
+    sha256 = "1yl25abf343yvd49nfsgxsz7jf956zrsi5n4xyqb5ldlp2hifk15";
+  };
+
+  buildInputs = [ cmake alsaLib libX11 ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp {m2o,o2m} $out/bin/
+  '';
+
+  meta = {
```

Comment:
It would be nice if you could use `with stdenv.lib;` here and save the copy/paste below.